### DISCUSSION TITLE
How to disable xray on video recording

### DISCUSSION THREAD

**Discussion by bsolyom** [akiver/cs-demo-manager#715] [Q&A] [answered]
Hi!
I want to disable Xray when recording a demo video, what parameter should I use here:
![image](https://github.com/akiver/cs-demo-manager/assets/63775925/9da32e06-a023-4ad9-b3b7-49a27ab10d13)
?

**Comment by akiver** [comment, marked as answer]
The command is `spec_show_xray 0`.
You have to paste it in the CFG input of the first sequence, example:

![xray](https://github.com/akiver/cs-demo-manager/assets/1842524/ccb27b33-4a07-4aba-b43f-cc9b1171cd6c)

I think I will add a video option to toggle it.

**Reply by bsolyom** [reply]
Thank you!
